hardhat.config.ts

```diff
@@ -102,16 +98,14 @@ const config: HardhatUserConfig = {
   networks: {
     'bsc-testnet': {
       url: 'https://bsc-testnet.blockpi.network/v1/rpc/public',
-      chainId: 97,
+      chainId: 91,
```

Comment:
```suggestion
      chainId: 97,
```
BSC testnet is [97](https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-97.json)